content/doc/developer/security/index.adoc

```diff
@@ -126,6 +126,8 @@ By flow we mean the involved classes that will check your credentials for validi
 
 image:web_rest_flow.svg["Web UI and REST API flow", role=center]
 
+In the diagram above, each arrow indicates a way to authenticate.
```

Comment:
@rtyler Do you know how to add caption to pictures ? Not strictly necessary but could be a +1 :)